store/interfaces.json

```diff
@@ -23,6 +23,10 @@
   "product-quantity-selector": {
     "component": "ProductQuantitySelector"
   },
+  "product-addon-wrapper": {
```

Comment:
Use a more descriptive name 

`product-add-to-list-button` or something like that